sdk/python/setup.py

```diff
@@ -77,7 +77,8 @@ def find_version(*file_path_parts):
     name=NAME,
     version=find_version('kfp', '__init__.py'),
     description='KubeFlow Pipelines SDK',
-    author='google',
+    author='Google',
```

Comment:
Should we update the author here to `kubeflow authors` similar to https://github.com/kubeflow/pipelines/issues/5617

Reply:
done!